index.md

```diff
@@ -5,4 +5,83 @@ title: issuehunter
 
 # Introduction
 
-What a wonderful project!
+Issuehunter is a decentralized application that enables Open Source crowdfunding
+on the Ethereum blockchain.
+
+It is a new way to improve Open Source development lifecycle by making it
+transparent and deterministic.
+
+It makes it easier for supporters to funds projects they love and for Open
+Source contributors to collect rewards for their work.
+
+The dapp is composed by three main elements:
+
+* a set of smart contracts deployed on the Ethereum blockchain
+* a service that helps verifying software patches submitted
+* a web based directory to discover funded issues that are still waiting to be
+  resolved
```

Comment:
Do we also have integration (an automatic comment or something like that) with github (and others to come later) to show directly on the issue page that there is an active bounty campaign on Issuehunter?